### ISSUE TITLE
ClickMeButton_Click missing in instruction 18

### ISSUE THREAD

**Issue by sjharper79** [markjprice/apps-services-net7#29]
Chapter: 18
Page Number: 666
Section Title: Adding shell navigation and more content pages
Problem to fix: In step 18, the instructions state "Modify the Button element to give it a name of ClickMeButton and a new event handler for its Clicked event, as shown in Figure 18.13." In the figure, the event handler name is blank. There is and intellisense popup that says "<New Event Handler>" but no actual event handler name. 
Suggested solution: Add the name to step 18 and update the image to include the name. Pressing Ctrl + Space autofills the name "ClickMeButton_Clicked" which is different from the name shown in step 20, which is "ClickeMeButton_Click."

**Comment by markjprice** [comment]
Yes, it is blank in the screenshot to show the *<New Event Handler>* text which is how you add a new event handler for its `Clicked` event. Only after you press Enter to select that will it then add the text inside the "". This is as described in the toolip, "Bind event to a newly created method called `ClickMeButton_Clicked`. Use `Go To Definition` to navigate to the newly created method." Alternatively, you can use your technique of pressing Ctrl + Space. Both techniques work.

In Step 20, you are correct that the method name is wrong in the book and should be `ClickMeButton_Clicked`. I have added this to the errata here: https://github.com/markjprice/apps-services-net7/blob/main/docs/errata/errata.md#page-667---adding-shell-navigation-and-more-content-pages, and here for 2nd edition: https://github.com/markjprice/apps-services-net8/blob/main/docs/errata/errata.md#page-726---adding-shell-navigation-and-more-content-pages
